.changeset/nine-falcons-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
```

Comment:
This is technically a new feature, so should be a `minor` change.

Reply:
OK, I thought as a patch it can be released independently of the upcoming 3.2. The feature is included in 3.2 anyway, so I can close this PR.